ale_linters/xml/xmllint.vim

```diff
@@ -0,0 +1,70 @@
+" Author: q12321q <q12321q@gmail.com>
+" Description: This file adds support for checking XML code with xmllint.
+
+" CLI options
+let g:ale_xml_xmllint_executable = get(g:, 'ale_xml_xmllint_executable', 'xmllint')
+let g:ale_xml_xmllint_options = get(g:, 'ale_xml_xmllint_options', '--noout')
+
+function! ale_linters#xml#xmllint#GetCommand(buffer) abort
+    return printf('%s %s -',
+    \   g:ale_xml_xmllint_executable,
```

Comment:
Call your executable callback here, and apply `ale#Escape` to your executable, in case your executable path contains spaces or other special characters.